neurom/features/__init__.py

```diff
@@ -64,15 +66,30 @@ def _flatten_feature(feature_shape, feature_value):
     return reduce(operator.concat, feature_value, [])
 
 
-def _get_neurites_feature_value(feature_, obj, neurite_filter, kwargs):
+def _get_neurites_feature_value(feature_, obj, neurite_filter, use_subtrees, **kwargs):
     """Collects neurite feature values appropriately to feature's shape."""
     kwargs.pop('neurite_type', None)  # there is no 'neurite_type' arg in _NEURITE_FEATURES
-    return reduce(operator.add,
-                  (feature_(n, **kwargs) for n in iter_neurites(obj, filt=neurite_filter)),
-                  0 if feature_.shape == () else [])
+
+    return reduce(
+        operator.add,
```

Comment:
equivalent to `sum()`, no?

Reply:
I guess I'm used to using `sum()` for all sorts of application of the `+` operator.  I find `reduce` requires more thinking; whatever you prefer

Reply:
As we discussed, let's revisit this in the other PR for the refactoring of this module.